src/nvim/decoration_provider.c

```diff
@@ -34,9 +34,9 @@ static kvec_t(DecorProvider) decor_providers = KV_INITIAL_VALUE;
 
 static void decor_provider_error(DecorProvider *provider, const char *name, const char *msg)
 {
-  const char *ns_name = describe_ns(provider->ns_id, "(UNKNOWN PLUGIN)");
-  ILOG("error in provider %s.%s: %s", ns_name, name, msg);
-  msg_schedule_semsg_multiline("Error in decoration provider %s.%s:\n%s", ns_name, name, msg);
+  const char *ns = describe_ns(provider->ns_id, "(UNKNOWN PLUGIN)");
+  ILOG("Error in decoration provider %s callback %s:\n%s", ns, name, msg);
+  msg_schedule_semsg_multiline("Error in decoration provider %s callback %s:\n%s", ns, name, msg);
```

Comment:
let's choose either "provider" or "callback". redundant to say both.

```suggestion
  ILOG("Error in decoration provider \"%s\" (ns=%s):\n%s", name, ns, msg);
  msg_schedule_semsg_multiline("Error in decoration provider \"%s\" (ns=%s):\n%s", name, ns, msg);
```